RealmSwift/Combine.swift

```diff
@@ -57,6 +57,18 @@ extension ObjectKeyIdentifiable {
     }
 }
 
+/// :nodoc:
+@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+extension ObjectKeyIdentifiable where Self: ProjectionObservable {
+    /// A stable identifier for this projection.
+    public var id: UInt64 {
+        // UInt64(bitPattern:) to properly cast negative Int64 values
+        // It will use the same memory representation, the numeric value
+        // may be different which is fine in this case
+        UInt64(withUnsafePointer(to: self, { UInt(bitPattern: $0) }))
```

Comment:
This is not a valid conformance to `Identifiable`. It needs to use the object key from the projected object.

Reply:
fixed